### ISSUE TITLE
Answers not shown in printable worksheet

### ISSUE THREAD

**Issue by msgj603** [numbas/Numbas#907]
When selecting "answer sheets" in the printable worksheet theme, no answers are appearing for me. Please see screenshot below:

![image](https://user-images.githubusercontent.com/5709336/164453217-fe817f54-9937-4022-a7fc-3bf636b82afd.png)

**Comment by christianp** [comment]
The "anwer sheets" option just shows the "Advice" section of each question. Have you put anything in there?
Can you give me a link to the exam on the numbas editor?

**Comment by msgj603** [comment]
Sorry, I thought it showed the answers as defined in "correct answer" box. Am I just imagining that it used to do that?!

**Comment by msgj603** [comment]
Yes it is showing the advice, so it seems to be behaving as expected.
